core/src/main/resources/jenkins/management/Messages.properties

```diff
@@ -56,3 +56,6 @@ NodesLink.Description=Add, remove, control and monitor the various nodes that Je
 ShutdownLink.DisplayName_prepare=Prepare for Shutdown
 ShutdownLink.DisplayName_cancel=Cancel Shutdown
 ShutdownLink.Description=Stops executing new builds, so that the system can be eventually shut down safely.
+
+EditDescripionLink.DisplayName=Edit Description
```

Comment:
This text is a bit vague IMHO. Highlighted text does not explain which description is going to be edited